packages/codegen-ui/lib/generate-form-definition/form-to-component.ts

```diff
@@ -13,66 +13,86 @@
   See the License for the specific language governing permissions and
   limitations under the License.
  */
-import { SchemaModel, ModelFields, isGraphQLScalarType } from '@aws-amplify/datastore';
 import {
   StudioComponent,
-  StudioForm,
   StudioComponentChild,
-  StudioComponentProperty,
-  DataStoreCreateItemAction,
-  DataStoreUpdateItemAction,
-  FixedStudioComponentProperty,
+  FormDefinition,
+  FormDefinitionElement,
+  FormStyleConfig,
+  StudioComponentProperties,
+  StudioFormStyle,
 } from '../types';
 
-// map the datastore schema fields into form fields
-export const mapFieldsToForm = (fields: ModelFields) => {
-  const formFields: StudioComponentChild[] = [];
-  Object.entries(fields).forEach(([fieldName, fieldValue]) => {
-    // TODO: expand studio component child to also support other non text fields
-    if (isGraphQLScalarType(fieldValue.type) && !fieldValue.isArray) {
-      formFields.push({
-        name: `${fieldName}Field`,
-        componentType: 'TextField',
-        properties: {
-          name: {
-            value: fieldName,
-          },
-          label: {
-            value: fieldName,
-          },
-          placeholder: {
-            value: `${fieldValue.type}`,
-          },
-          ...(fieldValue.isRequired && {
-            required: {
-              value: 'true',
-              type: 'boolean',
-            },
-          }),
-        },
-      });
-    }
-  });
+const DefaultStudioFormStyles = {
+  horizontalGap: '15px',
```

Comment:
You will find that the default styles are included in the form definition if there's no styles on StudioForm, so this can be removed.